sdk/eventhub/Azure.Messaging.EventHubs.Processor/perf/Scenarios/ProcessorReceiveTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Processor.Perf.Infrastructure;
+
+namespace Azure.Messaging.EventHubs.Processor.Perf.Scenarios
+{
+    public class ProcessorReceiveTest : ProcessorTest<ProcessorOptions>
+    {
+        public ProcessorReceiveTest(ProcessorOptions options) : base(options)
+        {
+            EventProcessorClient.ProcessEventAsync += ProcessEventAsync;
+            EventProcessorClient.ProcessErrorAsync += ProcessErrorAsync;
+        }
+
+        private Task ProcessEventAsync(ProcessEventArgs arg)
+        {
+            return EventRaisedAsync();
+        }
+
+        private Task ProcessErrorAsync(ProcessErrorEventArgs arg)
+        {
+            throw arg.Exception;
```

Comment:
Error handling during perf tests is tricky.  Our typical approach is that any error not handled by the SDK by default should fail-fast since it might impact the perf results.  However, errors that are handled by default (like HTTP retries) are ignored.

I've never had the ProcessErrorAsync() actually raised during any of my perf tests, so I'm inclined to keep this fail-fast for now, and we can revisit if it starts impacting too many perf tests.

Another option could be to print the exception to console but not re-throw, which might alert a human running the tests interactively, but would be ignored by automation.  So I'm hesitant to go this route as well.